src/components/Header/Header.story.js

```diff
@@ -0,0 +1,18 @@
+import { h } from 'preact';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+
+import { Header } from './Header';
+
+const backgroundImageUrl = require('../../assets/images/header-bg.png');
+
+storiesOf('Header', module)
+    .add('default', () => (
+        <Header
+            productName={text('productName', 'CSS Quickdraw')}
+            username={text('username', 'username')}
+            style={{
```

Comment:
Perhaps background could be set in the component style, so that `Header`s are configured with a `bgUrl` prop. On the other hand, there's apparent semantic distinction between "style" and "data props". Still there is no default background (and it doesn't make sense to have one), background is not optional so I wouldn't say it's styling or customization... 🤔

Reply:
We can try to make background b&w image and customize color with CSS (have no idea on top of my head right now how to do this)

Reply:
@kirilknysh Perhaps https://css-tricks.com/basics-css-blend-modes/

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/background-blend-mode - no IE support :(

Reply:
I'd drop IE support completely...

Reply:
Ooh, no Edge either 😿

Reply:
Multiple backgrounds then maybe

Reply:
Just to clarify, are you guys ok with style prop being passed down to header or you think `bgUrl` prop will be more verbose? I'd say having ability to pass`style` object has a benefit as you can override several background related properties alongside with changing backgroundUrl. As an example you might want to change background position/cover/attachment with different images

Reply:
I’d go with more strict ‘bgUrl’. Right now there’re no cases for overriding smth more than image

Reply:
see a31c3b3

Reply:
yay